src/main/client/src/utils/customTheme.ts

```diff
@@ -1,8 +1,14 @@
 import { extendTheme } from '@chakra-ui/react'
-import { mode } from '@chakra-ui/theme-tools'
+import { mode, createBreakpoints } from '@chakra-ui/theme-tools'
 
 // See more: https://chakra-ui.com/docs/theming/customize-theme
 const customTheme = extendTheme({
+  breakpoints: createBreakpoints({
+    sm: '30em',
+    md: '48em',
+    lg: '68em',
+    xl: '80em'
+  }),
```

Comment:
What happened to the old one? Seems fine, just curious.

Reply:
By default, the lg breakpoint was at 62em, but we wanted the middle column to be the first to wrap if the screen was smaller, and the first wrapping occurred at 68em, but this value depends on the width of the columns

Reply:
Only the lg was modified, but with your changes I think it should work without this.

Reply:
Yesssss it does. The 100% width of columns below md size will force the layout to wrap.